src/main/java/gov/usds/case_issues/model/CaseDetails.java

```diff
@@ -21,12 +20,12 @@
 
 	private TroubleCase rootCase;
 	private Collection<? extends CaseIssueSummary> issues;
-	private Collection<? extends CaseSnoozeSummary> snoozes;
+	private Collection<? extends CaseSnoozeSummaryFacade> snoozes;
```

Comment:
Is there a specific reason to change this from the interface to the implementation class? I'm thinking the answer is that the CaseSnoozeSummaryFaçade object has a different way of getting user information, which if so is a legit reason but maybe indicates we are going to have an uncomfortably tangled up set of related interface definitions sometime soon.

Reply:
`CaseSnoozeSummary `was changed to `CaseSnoozeSummaryFacade ` so that the case details endpoint returns the user information for snoozes

Reply:
Makes sense. So yeah, about those entangled database and API models/interfaces....